.gitignore

```diff
@@ -41,3 +41,6 @@ src/icons/skin/build-icons/icons/*.png
 
 # CMake build directory
 build/
+
+# clangd index
+.cache/clangd/
```

Comment:
This is generated by your own tool and not by this project. Therefore it should go into your own `~/.config/git/ignore` instead.